pubsub/api/Pubsub.Samples/PullMessagesCustomAsync.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START pubsub_subscriber_flow_settings]
+
+using Google.Cloud.PubSub.V1;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class PullMessagesCustomAsyncSample
+{
+    public async Task<List<string>> PullMessagesCustomAsync(string projectId,
+    string subscriptionId, bool acknowledge)
+    {
+        SubscriptionName subscriptionName = SubscriptionName.FromProjectSubscription(projectId,
+            subscriptionId);
+        var result = new List<string>();
+        SubscriberClient subscriber = await SubscriberClient.CreateAsync(
+            subscriptionName,
+            settings: new SubscriberClient.Settings()
+            {
+                AckExtensionWindow = TimeSpan.FromSeconds(4),
+                Scheduler = Google.Api.Gax.SystemScheduler.Instance,
```

Comment:
This probably is the default. Don't set it here, this is not a setting users will have to change, it's mostly there for testing purposes I believe.